### PR TITLE
Fixes visual bugs on protocols.io and home page [SCI-3634]

### PR DESCRIPTION
**Pull request by mlorb**
Jira ticket: [SCI-3634](https://biosistemika.atlassian.net/browse/SCI-3634)

### What was done
_Some visual bugs on protocols.io and home page were fixed._

### CODE REVIEW

File: app/assets/stylesheets/themes/scinote.scss [hunk 1]
```diff
@@ -154,6 +154,13 @@ body {
   width: 100% !important;
 }
 
+.input-group {
+  .form-control:not(:focus) {
+    -webkit-box-shadow: inset 0 1px 1px $color-white;
```
[the remark this answers is not in the document]
Comment (jbargu) [reply app/assets/stylesheets/themes/scinote.scss]: Use normal `box-shadow`.

File: app/views/shared/_navigation.html.erb [hunk 2]
```diff
@@ -53,7 +53,9 @@
         <% if current_user.teams.length > 0 %>
           <li id="team-switch">
             <div class="btn-group">
-              <button type="button" class="btn btn-default"><%= truncate_team_name(current_team.name) %></button>
+              <button type="button" class="btn btn-default" data-toggle="dropdown">
```
Comment (jbargu) [inline app/views/shared/_navigation.html.erb]: image
If you repeatedly click on it, the text start to disappear.
image

File: app/assets/stylesheets/themes/scinote.scss [hunk 3]
```diff
@@ -154,6 +154,13 @@ body {
...
+    box-shadow: inset 0 1px 1px $color-white;
```
[the remark this answers is not in the document]
Comment (mlorb) [reply app/assets/stylesheets/themes/scinote.scss]: I thought `-webkit-box-shadow` is needed for Safari. I can try to remove it and we will see...

[the remark this answers is not in the document]
Comment (aignatov-bio) [reply app/assets/stylesheets/themes/scinote.scss]: Can you please not change appearance of search field across whole SciNote. Just refactor search field in protocolio, remove button and add icon as overlay.

File: app/views/shared/_navigation.html.erb (hunk 2 above)
Comment (aignatov-bio) [reply app/views/shared/_navigation.html.erb]: I think this bug not correspond to this issue, and must be fixed in different task. I will create issue.

**Pull request merged**